platform-workflow-module/src/test/java/com/softicar/platform/workflow/module/workflow/task/WorkflowUserFieldTest.java

```diff
@@ -0,0 +1,111 @@
+package com.softicar.platform.workflow.module.workflow.task;
+
+import com.softicar.platform.common.date.DayTime;
+import com.softicar.platform.core.module.user.AGUser;
+import com.softicar.platform.workflow.module.demo.AGWorkflowDemoObject;
+import com.softicar.platform.workflow.module.test.AbstractTestObjectWorkflowTest;
+import com.softicar.platform.workflow.module.workflow.item.AGWorkflowItem;
+import com.softicar.platform.workflow.module.workflow.task.delegation.AGWorkflowTaskDelegation;
+import com.softicar.platform.workflow.module.workflow.task.delegation.AGWorkflowTaskDelegationLog;
+import com.softicar.platform.workflow.module.workflow.transients.field.WorkflowUserField;
+import java.util.Set;
+import org.junit.Test;
+
+public class WorkflowUserFieldTest extends AbstractTestObjectWorkflowTest {
+
+	private final AGWorkflowTask task;
+	private final AGWorkflowItem item;
+	private final AGUser user2;
+	private final AGWorkflowDemoObject object;
+	private final int TEST_AGE = 3;
+
+	public WorkflowUserFieldTest() {
+
+		item = insertWorkflowItem(rootNode);
+		this.task = insertWorkflowTaskOpen(user, item);
+
+		object = new AGWorkflowDemoObject() //
+			.setModuleInstance(item.getWorkflow().getModuleInstance())
+			.setName("TestObject")
+			.setWorkflowItem(item)
+			.save();
+
+		user2 = insertUser("Max", "Mustermann")//
+			.setEmailAddress("max@example.com")
+			.save();
+	}
+
+	@Test
+	public void testWithoutDelegation() {
+
+		WorkflowUserField<AGWorkflowDemoObject> workflowUserField = new WorkflowUserField<>(AGWorkflowDemoObject.WORKFLOW_ITEM);
+		Set<String> valueSet = workflowUserField.getValue(object);
+
+		assertEquals(1, valueSet.size());
+		String text = valueSet.iterator().next();
+		assertContains(user.getFirstName(), text);
+		assertContains(user.getLastName(), text);
+		assertContains("0 Days", text);
+	}
+
+	private void delegationSetup(boolean active) {
```

Comment:
- In unit tests, `private` methods should be below `@Test` methods.
- I think this method should be renamed to `insertDelegation`.

Reply:
fixed